spec/features/openid_connect/aal3_required_spec.rb

```diff
@@ -8,7 +8,7 @@
       it 'sends user to set up AAL3 auth' do
         user = user_with_2fa
 
-        visit_idp_from_ial1_oidc_sp_requesting_aal3(prompt: 'select_account')
+        visit_idp_from_ial1_oidc_sp_requesting_phishing_resistant(prompt: 'select_account')
```

Comment:
**question:** should we rename this file?

Reply:
renamed in https://github.com/18F/identity-idp/pull/7063/commits/a1b5c69ebba565a3b6d35035082e0ef01fbae1cd